src/js/actions/index.js

```diff
@@ -276,19 +276,29 @@ function fetchNumberIndicator(
       },
     });
   } else {
-    apiClient.get(url).then((res) => {
-      const indicatorData = res.data;
-      dispatch({
-        type: FETCH_NUMBERS,
-        payload: {
-          ...indicatorData,
-          id,
-          archived: indicatorConfig.archived,
-          enabled: indicatorConfig.enabled,
-        },
-      });
+    dispatch({
+      type: FETCH_NUMBERS,
+      payload: {
+        id,
+        title: 'Loading...',
+        type: 'loadingNumber',
+        archived: indicatorConfig.archived,
+        enabled: indicatorConfig.enabled,
+      },
     });
   }
+  apiClient.get(url).then((res) => {
```

Comment:
Previously this was triggered only if this was `indicatorConfig.enabled` true. Do you want to trigger `get` regardless of this?

Reply:
@awalkowiak oh no you are right, it's a mistake, I change that right now !